stdlib/file/src/main/ballerina/src/file/file_info.bal

```diff
@@ -26,6 +26,13 @@ public type FileInfo object {
     boolean dir;
     string path;
 
+    # Creates a FileInfo object.
+    #
+    # + name - Name of the file
+    # + size - Size of the file (in bytes)
+    # + modifiedTime - Time during which the file was modified last
```

Comment:
Can't we simply change this to `The last modified time of the file` ?

Reply:
Changed.